packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -524,16 +524,19 @@ class StackViewport extends Viewport implements IStackViewport {
    * @param interpolationType - Changes the interpolation type (1:linear, 0: nearest)
    * @param rotation - image rotation in degrees
    */
-  public setProperties({
-    voiRange,
-    invert,
-    interpolationType,
-    rotation,
-  }: StackViewportProperties = {}): void {
+  public setProperties(
+    {
+      voiRange,
+      invert,
+      interpolationType,
+      rotation,
+      suppressEvents,
```

Comment:
I think suppressEvents should be another argument, not part of the StackViewportProperties object. This makes it look like a property, which it is not.

Reply:
Agreed, can keep the API softfork-ey by putting it as the second argument and defaulting it to false.